docs/develop/typescript/failure-detection.mdx

```diff
@@ -1,164 +1,152 @@
-const { greet } = proxyActivities<typeof activities>({
+const { myActivity } = proxyActivities<typeof activities>({
   scheduleToCloseTimeout: '5m',
   // startToCloseTimeout: "30s", // recommended
   // scheduleToStartTimeout: "60s",
-
-  retry: {
-    // default retry policy if not specified
-    initialInterval: '1s',
-    backoffCoefficient: 2,
-    maximumAttempts: Infinity,
-    maximumInterval: 100 * initialInterval,
-    nonRetryableErrorTypes: [],
-  },
 });
 ```
 
-### How to set an Activity Retry Policy {#activity-retries}
+## Activity Retry Policy {#activity-retries}
+
+**How to set an Activity Retry Policy using the Temporal TypeScript SDK**
 
 A Retry Policy works in cooperation with the timeouts to provide fine controls to optimize the execution experience.
 
 Activity Executions are automatically associated with a default [Retry Policy](/encyclopedia/retry-policies) if a custom one is not provided.
 
-To set Activity Retry Policies in TypeScript, pass [`ActivityOptions.retry`](https://typescript.temporal.io/api/interfaces/common.ActivityOptions#retry) to [`proxyActivities`](https://typescript.temporal.io/api/namespaces/workflow/#proxyactivities).
+To set an Activity's Retry Policy in TypeScript, use the [`ActivityOptions.retry`](https://typescript.temporal.io/api/interfaces/common.ActivityOptions#retry) property when creating the corresponding Activity proxy function using the [`proxyActivities()`](https://typescript.temporal.io/api/namespaces/workflow#proxyactivities) API.
 
 ```typescript
-// Sample of typical options you can set
-const { yourActivity } = proxyActivities<typeof activities>({
+const { myActivity } = proxyActivities<typeof activities>({
   // ...
   retry: {
-    // default retry policy if not specified
-    initialInterval: '1s',
-    backoffCoefficient: 2,
-    maximumAttempts: Infinity,
-    maximumInterval: 100 * initialInterval,
-    nonRetryableErrorTypes: [],
+    initialInterval: '10s',
+    maximumAttempts: 5,
   },
 });
 ```
 
-## How to Heartbeat an Activity {#activity-heartbeats}
+## Activity next Retry delay {#activity-next-retry-delay}
+
+**How to override the next Retry delay following an Activity failure using the Temporal TypeScript SDK**
+
+The time to wait after a retryable Activity failure until the next retry is attempted is normally determined by that Activity's Retry Policy.
+However, an Activity may override that duration when explicitly failing with an [`ApplicationFailure`](https://typescript.temporal.io/api/classes/common.ApplicationFailure) by setting a next Retry delay.
+
+To override the next Retry delay for an `ApplicationFailure` thrown by an Activity in TypeScript, provide the [`nextRetryDelay`](https://typescript.temporal.io/api/interfaces/common.ApplicationFailureOptions#nextretrydelay) property on the object argument of the [`ApplicationFailure.create()`](https://typescript.temporal.io/api/classes/common.ApplicationFailure#create) factory method.
```

Comment:
"in Typescript" is already established by the heading.
Overall, this paragraph is simply restating what the code already says.  I can't find any value in it other than the links, which would be better and more succinctly provided IMO in a "see also" links list.

I also prefer how the java example incorporates attempt count via a more real world example, explaining how to do backoff.